hooks/blueprint

```diff
@@ -70,9 +71,16 @@ if want_feature "shield"; then
   echo >&2 "The 'shield' feature is no longer supported.  Instead, please add the"
   echo >&2 "shield agent to your runtime configuration."
 fi
+
 if want_feature "azure"; then
   echo >&2 "The 'azure' feature is no longer necessary - azure CPI will be detected"
   echo >&2 "automatically at deployment time."
 fi
 
+if want_feature "vault"; then
+  merge+=( "manifests/addons/vault.yml" )
+fi
```

Comment:
The vault feature should only be active if the `full` feature flag is on; since workers-only (satellite) deployments don't have ATC.